lib_test/bap_trace/test_trace.ml

```diff
@@ -0,0 +1,281 @@
+open OUnit2
+open Core_kernel.Std
+open Or_error
+
+open Bap_types.Std
```

Comment:
no,no,no, don't do this! All tests depend only on `Bap.Std`, and are external. 

P.S., you've made the same error on the image test, that always breaks the compilation with inconsistent assumptions.